app/livechat/client/views/app/tabbar/visitorInfo.js

```diff
@@ -298,13 +298,20 @@ Template.visitorInfo.onCreated(function() {
 		}
 	});
 
+	const loadRoomData = async (rid) => {
+		const { room } = await APIClient.v1.get(`rooms.info?roomId=${ rid }`);
+		this.visitorId.set(room && room.v && room.v._id);
+		this.departmentId.set(room && room.departmentId);
+		this.tags.set(room && room.tags);
+		this.room.set(room);
+	};
+
 	if (rid) {
-		this.subscribe('livechat:rooms', { _id: rid });
-		this.autorun(() => {
-			const room = LivechatRoom.findOne({ _id: rid });
-			this.visitorId.set(room && room.v && room.v._id);
-			this.departmentId.set(room && room.departmentId);
-			this.tags.set(room && room.tags);
+		this.autorun(async () => {
+			const action = this.action.get();
+			if (action === undefined) {
+				await loadRoomData(rid);
```

Comment:
no need to `await` since there is no code after the call
```suggestion
				loadRoomData(rid);
```